WordPress/src/main/java/org/wordpress/android/ui/main/WPMainActivity.java

```diff
@@ -469,30 +466,25 @@ && getIntent().getExtras().getBoolean(ARG_CONTINUE_JETPACK_CONNECT, false)) {
             UpdateTokenPayload payload = new UpdateTokenPayload(authTokenToSet);
             mDispatcher.dispatch(AccountActionBuilder.newUpdateAccessTokenAction(payload));
         } else if (getIntent().getBooleanExtra(ARG_SHOW_LOGIN_EPILOGUE, false) && savedInstanceState == null) {
-            canShowAppRatingPrompt = false;
```

Comment:
My perception is that the numerous `canShowAppRatingPrompt = false` disabled the prompt for various contexts. E.g., deep links to the login epilogue. Do we believe these guards are not longer necessary?

Reply:
The variable `canShowAppRatingPrompt` controlled only whether to show our custom rating dialog, which is dropped in this PR. It didn't have any role in showing the Google in-app review dialog.